Week0/markdown0.md

```diff
@@ -0,0 +1,11 @@
+# Benefits Automation 
+*it uses script to automate test scenarios.
+*scennarios ca be  executed repeatedly
+*The scripts cn be run on any platform or the browser
+* Can be started in the build phase of the SLDLC and then completed at the after  the functionality has been developed during the test phase
+*its a black box testing
+
+#Benefits of Manual 
+testing
+*manual testing is done at the Test phase of the SDLC
+*its a type of black box testing
```

Comment:
Tbf everything here looks sound to me obviously not an expert but it makes sense